### PR TITLE
motivation and evidence

### PR DESCRIPTION
**Pull request by cubap**
from https://github.com/cubap/Genealogger

I added evidence handling and created an attribute for recording the motivation.

### CODE REVIEW

**Comment by cubap** [comment]
fixes #16

File: js/deer-render.js [hunk 1]
```diff
@@ -133,6 +133,23 @@ DEER.TEMPLATES.prop = function (obj, options = {}) {
     }
 }
 
+/**
+ * Get a certain property from an object and return it formatted as HTML to be drawn.
```
Comment (thehabes) [inline js/deer-render.js:137]: Whoops, wrong documentation.  C & P problem, most likely.

File: js/deer-config.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ export default {
     LABEL: "title", // attribute, alternate label for properties
     CONTEXT: "deer-context", // attribute, JSON-LD @context, scoped
     ATTRIBUTION: "deer-creator", // attribute, Web Annotation `creator`, scoped
+    MOTIVATION: "deer-motivation", // attribute, Web Annotation `motivation`, scoped
```
Comment (thehabes) [inline js/deer-config.js:9]: I don't see a change for adding evidence here...

Upon viewing the file, I found it, it must have already been in there.  However `EVIDENCE` is `nv-evidence`.  I expected `deer-evidence`.  Is this on purpose?

File: js/deer-render.js (hunk 1 above)
Comment (thehabes) [inline js/deer-render.js:137]: Whoops, wrong documentation.

File: js/deer-config.js (hunk 2 above)
Comment (thehabes) [inline js/deer-config.js:9]: I didn't see a change for evidence in the GitHub interface.  Upon viewing the file, I found it, so it must have already been there.  However, `EVIDENCE` is `nv-evidence`.  Is this on purpose?

Comment (cubap) [reply js/deer-config.js:9]: We can have more of a conversation around this, but `evidence` is not a DEER concept, more of an Eventities one. We can add it to DEER without too much trouble (and perhaps less than creating a context for Eventities) but I didn't want to shoehorn it into DEER or RERUM unilaterally.

It has always been there and we've never talked about it.

**Pull request merged**